crates/release_plz_core/src/git/backend.rs

```diff
@@ -726,6 +833,19 @@ impl GitClient {
     }
 }
 
+fn validate_labels(labels: &Vec<String>) -> anyhow::Result<()> {
+    for l in labels {
+        if l.len() > 50 {
+            anyhow::bail!("Failed to add label `{l}`. It exceeds maximum length of 50 characters.");
+        }
+
+        if l.trim().is_empty() {
+            anyhow::bail!("Failed to add label. Empty labels are not allowed.");
+        }
+    }
+    Ok(())
+}
```

Comment:
can we move the validation when we read the configuration? So that we fail earlier.

Reply:
I moved this to the release_pr command section